### ISSUE TITLE
Unable to login with default information + image uploads fail

### ISSUE THREAD

**Issue by ZuluWhiskey** [babybuddy/babybuddy#684]
Tried setting this up with docker and I'm getting some off behaviour - I'm sure I've done something wrong but I have no idea what that is!

## Docker version
24.0.4, build 3713ee1

## Ubuntu version
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal

## Docker Compose YAML
```
version: '2.1'

services:
  babybuddy:
    image: lscr.io/linuxserver/babybuddy
    container_name: babybuddy
    environment:
      - TZ=Europe/London
      - PUID=981
      - PGID=981
      - DEBUG=1
    volumes:
      - ./config:/config
    ports:
      - 8000:8000
    restart: unless-stopped
```

## Issue
After pulling the container and starting it, I am unable to login with admin/admin. To get around this issue, I did a `chmod -r 777` to the `config` directory and this then allowed me to log in. So this looks like some sort of permissions issue but I'm not sure what/why

Once logged in, I tried to add a child but for this I also got an error - `500 server error` without debug enabled, with debug enabled I get `module 'PIL.Image' has no attribute 'ANTIALIAS'`  - [See debug output here](https://github.com/babybuddy/babybuddy/files/12117744/babybuddy_image_error.txt)

I have lots of other containers running on my system without these sorts of issues so I'm assuming it's some quirk for this application that I've perhaps missed somewhere, so apologies in advance for my ignorance!

**Comment by cdubz** [comment]
I’ll give this a test when I can but in the meantime you may want to repost this over at https://github.com/linuxserver/docker-babybuddy/issues. The LSIO folks who manage the Baby Buddy image are very helpful… and much more knowledgeable about Docker than I 😃

**Comment by aptalca** [comment]
We hit the Pillow issue on another image as well. It seems the latest Pillow version deprecated `antialias` so the upstream projects would need to be updated for compatibility (or the pillow version pinned to older).

**Comment by cdubz** [comment]
Ah thanks @aptalca. I’ll take a look. Do you have a ref for where you saw this previously? Curious what version of Pillow to look for.

**Comment by aptalca** [comment]
Just this issue report for our beets image: https://github.com/linuxserver/docker-beets/issues/107

Pillow's changelog also mentions the removal of that constant: https://pillow.readthedocs.io/en/stable/releasenotes/10.0.0.html#constants

**Comment by ZuluWhiskey** [comment]
> I’ll give this a test when I can but in the meantime you may want to repost this over at https://github.com/linuxserver/docker-babybuddy/issues. The LSIO folks who manage the Baby Buddy image are very helpful… and much more knowledgeable about Docker than I 😃 

Forgot to actually reply - Thanks very much, I've raised an issue over there too but can obviously see that you're already discussing here too 👍

**Comment by cdubz** [comment]
Ok fixing this for now by locking Pillow at 9.5.0.

See https://github.com/matthewwithanm/django-imagekit/issues/566 and https://github.com/matthewwithanm/pilkit/issues/64. It looks like both of those packages may be abandoned but we'll give it some time before looking for a replacement. I'll open a separate issue to track that.

I'll cut a new 2.0.4 release this this fix (and the login fail should be fixed in 2.0.3).
